src/main/java/services/tasklist/TaskList.java

```diff
@@ -0,0 +1,160 @@
+package services.tasklist;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import command.CommandType;
+import services.bizerrors.EmptyArgumentException;
+import services.bizerrors.IndexOutOfRangeException;
+import services.bizerrors.JarvisException;
+import services.bizerrors.SaveToFileException;
+import services.tasklist.tasks.Deadline;
+import services.tasklist.tasks.Event;
+import services.tasklist.tasks.Task;
+import services.tasklist.tasks.Todo;
+
+public class TaskList implements ITaskList {
+    /** The list of tasks. */
+    protected List<Task> taskList;
```

Comment:
Perhaps use plural form for variable name since it is a collection of objects (A list of tasks)? 

An example from the [SE Coding Standard Guide](https://se-education.org/guides/conventions/java/intermediate.html#naming) is:
```
Collection<Point> points;
```
So for your case, perhaps you can use ```List<Task> tasks```, if it makes sense.

Reply:
Ok! I'll take note of it and change it soon!